.gitignore

```diff
@@ -0,0 +1,4 @@
+.terraform.lock.hcl
+.terraform
+terraform.tfstate
+terraform.tfstate.backup
```

Comment:
Please add a trailing newline to the end of the file so there is no ⛔ sign here and in each file.
This is important since some Linux utilities may read the file without a trailing newline incorrectly